javascript/javascript3/week2/currency_converter/styles.css

```diff
@@ -0,0 +1,7 @@
+body,
+.wrapper {
```

Comment:
Do you need the same style defined for both body and div with the class wrapper?

Reply:
It was a simple way to center the widget on a page and in the same time center the fields inside the widget.
But you are right - I should have added more styling options (like dark mode, fields styles etc.)
Hope to revisit this after I'm done with databases...